kinder/ci/workflows/ha-cert-copy-with-external-etcd.yaml

```diff
@@ -0,0 +1,119 @@
+version: 1
+summary: |
+  This workflow implements a sequence of tasks used for creating a HA
+  cluster with an external etcd cluster.
+vars:
+  kubernetesVersion: v1.14.1
+  controlPlaneNodes: 2
+  baseImage: kindest/base:v20190403-1ebf15f
+  image: kindest/node:test
+  clusterName: kinder-ha-external-etcd
+tasks:
+- name: pull-base-image
+  description: |
+    pulls kindest/base image with docker in docker and all the prerequisites necessary for running kind(er)
+  cmd: docker
+  args:
+    - pull
+    - "{{ .vars.baseImage }}"
+- name: add-kubernetes-versions
+  description: |
+    creates a node-image-variant by adding Kubernetes version "kubernetesVersion"
+    to be used when executing "kinder do kubeadm-init"
+  cmd: kinder
+  args:
+    - build
+    - node-image-variant
+    - --base-image={{ .vars.baseImage }}
+    - --image={{ .vars.image }}
+    - --with-init-artifacts={{ .vars.kubernetesVersion }}
+    - --loglevel=debug
+  timeout: 10m
+- name: create-cluster
+  description: |
+    create a set of nodes ready for hosting the Kubernetes cluster
+  cmd: kinder
+  args:
+    - create
+    - cluster
+    - --name={{ .vars.clusterName }}
+    - --image={{ .vars.image }}
+    - --control-plane-nodes={{ .vars.controlPlaneNodes }}
+    - --worker-nodes=1
```

Comment:
conformance needs at least 2 workers.

Reply:
Interesting :thinking:, they passed here with one, will change nevertheless.

Reply:
there was a discussion some time ago by k8s conformance.

these comment covers some of that:
https://github.com/kubernetes/kubernetes/issues/69601#issuecomment-428755808
https://github.com/kubernetes/kubernetes/issues/69601#issuecomment-455620205

this test should still require two nodes:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apps/daemon_set.go#L386
but they are Serial tests and we don't run them in the kinder jobs.

we should have 2+ just in case.

Reply:
Thank you a lot for the detailed reasoning. Will update tomorrow.